### ISSUE TITLE
swc-plugin inserts imports before `"use client"`

### ISSUE THREAD

**Issue by hehex9** [lingui/swc-plugin#21]
```
The "use client" directive must be placed before other expressions.
```
ref: https://beta.nextjs.org/docs/rendering/server-and-client-components#convention

## Minimal reproducible example

**input.js**
```js
'use client'
import {t} from '@lingui/macro'
import foo from 'bar'
t`Text`
```

**package.json**
```json
{
  "devDependencies": {
    "@lingui/swc-plugin": "0.2.0",
    "@swc/cli": "0.1.62",
    "@swc/core": "1.3.36"
  }
}
```

## Actual
```js
import { i18n } from "@lingui/core";
"use client";
import foo from "bar";
i18n._("Text");
```

## Expected
```js
"use client";
import { i18n } from "@lingui/core";
import foo from "bar";
i18n._("Text");
```

**Comment by thekip** [comment]
Thanks for reporting, i will take a look

**Comment by andrii-bodnar** [comment]
Fixed in the [0.2.1](https://github.com/lingui/swc-plugin/releases/tag/0.2.1)
